### PR TITLE
feat: Adding AdaBoost Classification

### PR DESCRIPTION
**Pull request by HaibinLai**
Added AdaBoost Classification with FlAML framework.

# Testing

## Choosing dataset and Classification ML workflow
Select data:
d66255f84c9bfce4410cdbcbf4c6c186

Here we select column 2 "Label" for Y set, column [3,12] as X set.
9fa29828e8490f6a1f8c1ce66220e025

Then choose mode2: Classification. Here we can see "AdaBoost" in label 11:
9de4f361e67ae3c786edade1b95eed9e

In feature engineering part we get the data.
4349bd32d6a2fefa531a7fe5b63628d1

## AdaBoost in non-autoML workflow

Select parameters for AdaBoost. Here we expose 3 hyperparameter in the model: *N estimator* for how many decision trees we are going to use, *Learning rate* for the model, *max_depth* for the depth of the decision trees used in AdaBoost.

4477471d0b6bd9f34de78579ba3cc823

Training and collecting result:

b9b146441a0ed6e5711e757fc5d7c7de

d8881aee06faa8221cfa89a4c98b28fe

## AutoML Workflow

Select AutoML here:
image

Auto tuning:
image
image

image

Final Result:
392a86a04e416eed135a71c83a1089d5

### CODE REVIEW

**Comment by HaibinLai** [comment]
## Choosing dataset and Classification ML workflow
image
image
image

## AdaBoost in non-autoML workflow
image
image
image
image
image
image

AutoML
image
image
image

final Result
image
